deployments/deploy-ssh.sh

```diff
@@ -17,6 +17,15 @@ if [ -z "${DASH_UID}" ]; then
 fi
 
 SECURITY_CONTEXT=false
+
+# System checks https://github.com/portworx/torpedo/blob/86232cb195400d05a9f83d57856f8f29bdc9789d/tests/common.go#L2173
+# should be skipped from AfterSuite() if this flag is set to true. This is to avoid distracting test failures due to
+# unstable testing environments.
+TORPEDO_SKIP_SYSTEM_CHECKS=false
+if [[ ! -z "${TORPEDO_SKIP_SYSTEM_CHECKS}" ]]; then
+    TORPEDO_SKIP_SYSTEM_CHECKS=true
```

Comment:
can we default it to false so that we do not need to update all our existing jobs

Reply:
The default is indeed false.